scheduler/src/cook/mesos/api.clj

```diff
@@ -354,6 +354,12 @@
    (s/optional-key :detailed) s/Bool
    (s/optional-key :partial) s/Bool})
 
+(def KillGroupsParams
+  "Schema for killing groups, allowing optionally for 'partial'
+  results, meaning that some uuids can be valid and others not"
+  {:uuid [s/Uuid]
+   (s/optional-key :partial) s/Bool})
```

Comment:
Am I missing something, or are we not using the `partial` flag? 

I'm not sure if we need it for kill, what do you think? 

In the CLI, when we implement `kill`, we will probably need to first query for the UUIDs, to make sure they don't exist on multiple clusters, since we don't want to kill on multiple clusters (the user will need to tell us which cluster). Since we've already queried, once we actually go to do the kill, we will know the UUID(s) exist(s) on the cluster we're doing the kill against.

Thoughts?

Reply:
The partial flag is checked in the `:exists?` method, but after discussing this offline with you and Paul, I went ahead and deleted this. I agree with both of your reasoning that users should be precise about what they want to delete.